catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -162,8 +161,8 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
                   workId={result.source.id}
                   image={{
                     contentUrl: result.src,
-                    width: 156,
-                    height: 156,
+                    width: result.width * 1.57,
```

Comment:
What does `1.57` stand for?

Reply:
ah, it should be 1.56, not 1.57
above, when creating `imagesWithDimentions` array, we create the images with the fixed height of 100px, but since the original was 156 in here, I multiplied the value by 1.57 to get close to the previous size

Reply:
Ah that makes sense! Could we leave a comment for legacy purposes? I think it was 156px because 100px was too small, not sure why that number exactly though 😅